march_hardware/src/encoder/Encoder.cpp

```diff
@@ -21,6 +21,21 @@ int32_t Encoder::getAngleIU(uint8_t byte_offset) const
   return return_byte.i;
 }
 
+double Encoder::getVelocityIU(uint8_t byte_offset) const
+{
+  if (this->slave_index_ == -1)
+  {
+    ROS_FATAL("Encoder has slaveIndex of -1");
+  }
+  union bit32 return_byte = get_input_bit32(this->slave_index_, byte_offset);
+  return return_byte.i / FIXED_POINT_TO_FLOAT_CONVERSION * TIME_PER_VELOCITY_SAMPLE;
```

Comment:
```suggestion
  return return_byte.i * TIME_PER_VELOCITY_SAMPLE / FIXED_POINT_TO_FLOAT_CONVERSION;
```
I somewhat prefer this order to avoid any ambiguity in order of calculations.

Reply:
Better yet is to add braces